src/providers/staleness-provider.js

```diff
@@ -35,7 +35,7 @@ export default class YamcsStalenessProvider {
     }
 
     subscribeToStaleness(domainObject, callback) {
-        return this.#openmct.telemetry.subscribeToStaleness(domainObject, callback);
+        return this.realtimeTelemetryProvider.subscribeToStaleness(domainObject, callback);
```

Comment:
lets add some coverage for staleness

Reply:
All right, I'll pull the staleness and fault management fixes out of this PR and we can do this in a separate one.